src/dataframe/dataframe.jl

```diff
@@ -772,6 +772,8 @@ Mutate `df` in place to retain only columns specified by `inds...` and return it
 
 Arguments passed as `inds...` can be any index that is allowed for column indexing
 provided that the columns requested in each of them are unique and present in `df`.
+In particular, you are allowed to use regular expressions,
```

Comment:
I'd avoid using "you" in docstrings. Better say "are supported" or "`inds` can be". Also use `@ref`?

Reply:
changed to "are supported".

AFAIK `@ref` will not work as `Not` etc. are not defined in DataFrames.jl and cross-package `@ref` does not work (at least as of present).